src/app/modules/onboarding/view.nim

```diff
@@ -36,6 +36,7 @@ QtObject:
 
   proc appLoaded*(self: View) {.signal.}
   proc accountLoginError*(self: View, error: string, wrongPassword: bool) {.signal.}
+  proc accountLoginSuccess*(self: View) {.signal.}
```

Comment:
I could add a `keyUID` param here for the keychain to store the secrets, wdyt @igor-sirotin @micieslak ?